src/vs/workbench/parts/terminal/electron-browser/terminalInstance.ts

```diff
@@ -585,16 +589,34 @@ export class TerminalInstance implements ITerminalInstance {
 		if (!this._shellLaunchConfig.executable) {
 			this._configHelper.mergeDefaultShellPathAndArgs(this._shellLaunchConfig);
 		}
-		this._initialCwd = this._getCwd(this._shellLaunchConfig, this._historyService.getLastActiveWorkspaceRoot('file'));
 
-		// Merge process env with the env from config
-		const envFromConfig = { ...process.env };
+		const lastActiveWorkspaceRootUri = this._historyService.getLastActiveWorkspaceRoot('file');
+		const lastActiveWorkspaceRoot = this._workspaceContextService.getWorkspaceFolder(lastActiveWorkspaceRootUri);
```

Comment:
Let's move this variable down to the other chunk to make it clear that it's not used in the following line.